File: app/src/main/java/com/example/testesantander/login/LoginResponse.java
package com.example.testesantander.login;

import com.google.gson.annotations.SerializedName;

import com.example.testesantander.model.Error;
import com.example.testesantander.domain.User;

public class LoginResponse {
    private Error error;

    @SerializedName("userAccount")
    private User user;

    public LoginResponse(Error error, User user) {
        this.error = error;
        this.user = user;
    }

    public Error getError() {
        return error;
    }

    public User getUser() {
        return user;
    }
}

File: App/data/src/main/java/com/santander/data/source/remote/entity/response/LoginResponse.kt
package com.santander.data.source.remote.entity.response

import com.google.gson.annotations.SerializedName

data class LoginResponse(

	@field:SerializedName("userAccount")
	val userAccount: UserAccountResponse,

	@field:SerializedName("error")
	val error: ErrorResponse? = null
)